pkg/commands/internal/vibra.go

```diff
@@ -0,0 +1,32 @@
+package internal
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+type FlagFunc[T any] func(name string, value T, usage string) *T
+
+type FlagPFunc[T any] func(name, shorthand string, value T, usage string) *T
+
+// Vibra adds a Cobra flag and binds it with Viper.
```

Comment:
Can't we just call this what it does so it gives some context even if you're not fully familiar with Viper and Cobra (which I guess this is some combination of)?

Suggestion:
- `AddFlagAndBind
- `AddAndBindFlag`

Also maybe I'm missing something but where is Cobra in this picture? Can't see it here or from the caller in `flagset`, seems to be using `pflag` which doesn't look to be a part of Cobra?

Reply:
Sorry to be a party 🎉 pooper 💩! If you like the name, keep it! I guess you won't read this code anyways if you're not familiar with the concepts and the comments explain the name!

Reply:
I'm not attached to this name :smile: it was just fun, so I will keep your suggestion in can be better for readability.